docs/source/processing-levels.md

```diff
@@ -0,0 +1,61 @@
+
+The `echopype` team is developing a clearly defined progression of data processing levels for active ocean sonar data, leveraging the accumulated experience from remote sensing and large-scale, long-term ocean and ecological observatories [7]–[10]. Data processing functions in `echopype` are clearly associated with specific Processing Level inputs and outputs, and when appropriate, will generate a `processing_level` dataset global attribute with entries such as "Level 1A", "Level 2B", "Level 4", etc.
+
+
+## Processing Levels and Sub-levels
+
+### Level 0 (L0)
+
+**Description:**  Raw data in vendor sensor format.
+
+- raw binary files. Associated metadata may be found in external files.
+
+### Level 1 (L1)
+
+**Description:** Raw data packaged with ancillary information and converted and standardized to an open convention and standard data formats. May be distributed as sets of individual converted files as originally segmented into arbitrary time ranges during sensor file creation or compiled into larger granules corresponding to logical deployment intervals.
+
+- **L1A**: Raw L0 data converted to a standardized, open format with geographic coordinates (latitude & longitude) processed and included. Includes other ancillary information extracted from sensor-generated L0 data or other external sources. May include environmental information such as temperature, salinity and pressure. Use of the SONAR-netcDF4 v1 convention is strongly recommended.
+- **L1B**: L1A data with quality-control steps applied, such as time-coordinate corrections that enforce strictly increasing, non-duplicate timestamps.
+
+### Level 2 (L2)
+
+**Description:** Calibrated acoustic quantities at raw data resolution, with spatial coordinates included (latitude, longitude and depth)
+
+- **L2A**: Volume backscattering strength (`Sv`) with interpolated latitude, longitude and depth coordinates. May incorporate addition information, such as  split beam angle
+- **L2B**: `Sv` L2A data with noise removal or other data filtering applied, including seafloor bottom removal.
+
+### Level 3 (L3)
+
+**Description:** Calibrated acoustic quantities regridded or aggregated to a common grid across channels. May include noise removal or other filtering.
+
+- `Sv` resampled to a common, high-resolution grid across channels
+- Mean Volume Backscattering Strength (`MVBS`)
+- `Sv` frequency difference across two channels resampled to a common grid
```

Comment:
I am not sure why the frequency difference between Sv from two channels is a data product. It is more of a processing step to classify?  Do you mean the masked Sv afterwards?

Reply:
I agree. I've made a suggestion to emphasize the mask application, and retaining frequency differencing only as a mask example. How does that look?

Reply:
I think that would work for this iteration. Let's discuss where masks should belong as data products -- i feel it may belong to L4, but we can reference other examples to determine.